### DISCUSSION TITLE
Sigma B88 (HD64016 / H16) ROMs dumped - looking to implement this system in MAME

### DISCUSSION THREAD

**Discussion by ethteck** [mamedev/discussions#84] [Q&A]
Hi! I've acquired and dumped 4 ROMs for a Sigma B88 system, which uses the HD641016 CPU (H16 architecture)

I see that MAME has seemingly partially implemented support for this CPU and at least a somewhat if not completely working disassembler. I was wondering if there was someone I could talk to in order to get help further adding support for this system. ajrhacker seems to be the one who originally implemented the support that is there, and they're also the main contributor to the project, so maybe they're too busy. but could someone else maybe help guide me in this process, or does anyone know anything about this system?

**Comment by Sonic-Amiga** [comment]
Hello! 3 years have passed, not sure if anyone would reply. But...
I've just acquired a B88 motherboard. I am a retro-computing enthusiast, just got some interesting "junk" for cheap on a local online marketplace.
 The condition is not known. I have no documentation, no schematic, no knowledge, nothing. I've just picked two datasheets: on the CPU and on insane 4K GPU from 1984 (!!!). I am super-excited!
 Not sure, however, when i get time to work on this machine. But i definitely want to.
